dotnet/Directory.Packages.props

```diff
@@ -7,10 +7,9 @@
   <ItemGroup>
     <PackageVersion Include="Azure.AI.Inference" Version="1.0.0-beta.1" />
     <PackageVersion Include="Microsoft.VisualStudio.Threading" Version="17.11.20" />
-    <PackageVersion Include="OpenAI" Version="2.0.0-beta.11" />
-    <PackageVersion Include="System.ClientModel" Version="1.1.0-beta.7" />
```

Comment:
Good catch, indeed we get this package by reference to OpenAI, not necessary here anymore.